app/js/controllers/service.js

```diff
@@ -3,12 +3,27 @@
 import { Controller } from 'components/fxos-mvc/dist/mvc';
 
 import Service from 'js/views/service';
+import CameraService from 'js/views/camera-service';
 
 export default class ServiceController extends Controller {
   main(id) {
```

Comment:
__question:__ Here is a big question :) What was your plan for choosing appropriate view for specific devices? At what stage should we choose what view we'd like to render? For now I'm requesting service here to know what view should I render and pass service to the view itself.